src/web/utils/mod.rs

```diff
@@ -0,0 +1,3 @@
+mod extensions;
+
+pub(crate) use self::extensions::InjectExtensions;
```

Comment:
I wanted to move it to a module since at the moment `web` contains all the routes, and having that merged with the other files could be confusing. I'm open to change it though.

Reply:
I thought only `web/routes.rs` had routes?